src/ru/nsu/ccfit/zuev/osu/game/GameObject.java

```diff
@@ -51,7 +51,9 @@ public boolean isStartHit(){
         return startHit;
     }
 
-    public void tryHit(float dt) {return;};
+    public boolean tryHit(float dt) {return false;};
 
     public PointF getPos() {return pos;};
+
+    public void forceMiss() {};
```

Comment:
Same as above.

```suggestion
    public abstract void forceMiss();
```

Reply:
this method is useless in many `GameObject`'s subclasses. if make it `abstract`, it will add some useless code.